### PR TITLE
Fixes to recurring transactions

### LINKED ISSUE #47: Make instance of recurring with different date

### PR DESCRIPTION
**Pull request by kalkins**
Fixes #47

### CODE REVIEW

**Review by nicomni — COMMENTED** [review]
This is only a namespace change, right?

**Comment by kalkins** [comment]
@NicolajNiels1 Partly, it changes the `day_delta` and `month_delta` fields on recurring transactions to `ìnterval` and `interval_type`, and allows instances of recurring transactions on different dates than set by those fields

**Comment by FredrikAugust** [comment]
@kalkins I tried to look at the merge conflict, but I'm afraid I'll break it. Think you could take a look?

And please ping me when you're done, so I can review it ASAP. I really want this in master ;)

File: transaction/models.py [hunk 1]
```diff
@@ -68,27 +80,30 @@ class RecurringTransaction(models.Model):
...
+        if self.interval_type == self.MONTH:
             freq = MONTHLY
-            interval = self.month_delta
-        else:
+        elif self.interval_type == self.DAY:
             freq = DAILY
-            interval = self.day_delta
 
-        occurences = rrule(freq, interval=interval, dtstart=self.start_date, until=end_date)
+        occurences = rrule(freq, interval=self.interval, dtstart=self.start_date, until=end_date)
```
Comment (FredrikAugust) [inline transaction/models.py]: Could perhaps replace `freq` with `self.interval_type`?

**Review by FredrikAugust — APPROVED** [review]

**Pull request merged**